core/cap-0062.md

```diff
@@ -0,0 +1,281 @@
+most Bucket merges.
+
+For Bucket merges, the newest version of a given key is always taken. At the bottom level, `HOT_ARCHIVE_LIVE` entries are dropped.
+The `HOT_ARCHIVE_LIVE` state indicates that the given key currently exists in the Live BucketList. Thus, any Hot Archive reference
+is out of date and can be dropped.
+
+The current Hot Archive is published to the History Archive via the "history" category on every checkpoint ledger.
+
+##### Hot Archive BucketList Depth and Spill Schedule
+
+The Hot Archive is guaranteed to experience less churn than the Live BucketList, given that the only events
+than can modify the Hot Archive BucketList are eviction and restoration. Thus, the spill schedule and depth
+of the Hot Archive BucketList should not match that of the Live BucketList. Long term, it is expected that
+the Hot Archive BucketList become significantly bigger than the Live BucketList (assuming State Archival
+for classic entry types and increased Soroban adoption).
+
+Specific configurations will need to be investigated, but a slower spill schedule is most likely optimal. A
+deeper BucketList (to accommodate the expected size difference between Live and Hot Archive) may not be
+necessary, as the spill schedule impacts the maximum "capacity" of a BucketList along with the depth.
+
+#### Changes to History Archives
+
+The `HistoryArchiveState` will be extended as follows:
+
+```
+version: number identifying the file format version
+server: an optional debugging string identifying the software that wrote the file
+networkPassphrase: an optional string identifying the networkPassphrase
+currentLedger: a number denoting the ledger this file describes the state of
+currentBuckets: an array containing an encoding of the live state bucket list for this ledger
+hotArchiveBuckets: an array containing an encoding of the hot archive bucket list for this ledger
+```
+
+Bucket files for the `hotArchiveBuckets` will be stored just as Bucket files are currently stored.
+
+#### Changes to LedgerHeader
+
+While the XDR structure of `LedgerHeader` remains unchanged, `bucketListHash` will be changed to the following:
+
+```
+header.bucketListHash = SHA256(liveStateBucketListHash, hotArchiveHash)
+```
+
+#### Meta
+
+Whenever a `PERSISTENT` entry is evicted (i.e. removed from the Live State BucketList and added to the Hot Archive),
+the entry key and its associated TTL key will be emitted via `evictedLedgerKeys`
+(this field has been renamed and was previously named `evictedTemporaryLedgerKeys`).
+
+Whenever an entry is restored via `RestoreFootprintOp`, the `LedgerEntry` being restored and its
+associated TTL will be emitted as a `LedgerEntryChange` of type `LEDGER_ENTRY_RESTORE`. Note that
+entries being restored may not have been evicted such that entries currently in the Live State BucketList
+can see `LEDGER_ENTRY_RESTORE` events.
```

Comment:
For this

> Note that entries being restored may not have been evicted such that entries currently in the Live State BucketList can see `LEDGER_ENTRY_RESTORE` events.

I'd like to make sure I understand it correctly: Are you saying that `LEDGER_ENTRY_RESTORE` does not necessarily mean restoration from eviction because live state may have an expired-yet-not-evicted entry that can be restored before it pops off the end of the BL?

If yes, I'd like to suggest a rewording:

```suggestion
Whenever an entry is restored via `RestoreFootprintOp`, the `LedgerEntry` being restored and its
associated TTL will be emitted as a `LedgerEntryChange` of type `LEDGER_ENTRY_RESTORE`. Note that
entries can be restored from both the Live State and Hot Archive BucketList, because archived entries 
continue to exist in Live State until they are evicted as part of natural BucketList growth. The emitted
meta does not distinguish between these.
```

Reply:
Your understanding is correct, I'll reword it.